docs/src/documentation/03-components/01-action/buttongroup/meta.yml

```diff
@@ -1,3 +1,5 @@
 title: ButtonGroup
 excerpt: Groups multiple related actions a user can take.
 type: tabs
+links:
+  - https://github.com/kiwicom/orbit/tree/master/packages/orbit-components/src/ButtonGroup/
```

Comment:
Just FYI, I think we can infer source links based on the `title`, it seems that every component link is `https://github.com/kiwicom/orbit/tree/master/packages/orbit-components/src/<title>` 🤷

Reply:
That's a fair point. Things like the primitives will be different, but the title can be the default. I'd leave it as it is until we have something actually implementing the links. I guess we don't actually have a task for that. I could add it to [ORBIT-1642](https://jira.kiwi.com/browse/ORBIT-1642) or create a new one.